brave/brave-ios
# Fix #4876: Remove clipboard checker for handling user referral installs.

## Summary of Changes


This pull request fixes #4876 

## Submitter Checklist:


## Test Plan:

1. Copy some random text to your clipboard
2. Fresh install the app
3. Verify that there is no screen that asks for your clipboard, the first screen should be 'Welcome to Brave - Let's go'

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
